templates/summit/index.html

```diff
@@ -0,0 +1,328 @@
+{% extends "summit/_base_summit.html" %}
+
+{% block title %}Ubuntu Summit 2024{% endblock %}
+
+{% block meta_description %}
+  The Ubuntu Summit is a showcase for the innovative and the ambitious. We invite all experts, builders, engineers, tinkerers, designers and dreamers to join us for an unforgettable three days.
+{% endblock %}
+
+{% block meta_image %}
+  https://assets.ubuntu.com/v1/9e91caf9-Canonical%20Ubuntu%20Summit%20Logo%202024%20with%20text.svg
+{% endblock %}
+
+{% block content %}
+
+  <header id="navigation" class="p-navigation is-dark">
+    <div class="p-navigation__row--25-75">
+      <div class="p-navigation__banner">
+        <div class="p-navigation__tagged-logo">
+          <a class="p-navigation__link" href="#">
+            <div class="p-navigation__logo-tag">
+              <img class="p-navigation__logo-icon"
+                   src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg"
+                   alt="" />
+            </div>
+            <span class="p-navigation__logo-title">Ubuntu Summit</span>
+          </a>
+        </div>
+        <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+        <a href="#navigation-closed"
+           class="p-navigation__toggle--close"
+           title="close menu">Close menu</a>
+      </div>
+      <nav class="p-navigation__nav" aria-label="Ubuntu summit navigation">
+        <ul class="p-navigation__items">
+          <li class="p-navigation__item is-selected">
+            <a class="p-navigation__link" href="#">About</a>
+          </li>
+          <li class="p-navigation__item">
+            <a class="p-navigation__link" href="https://events.canonical.com/event/51/registrations/">Register</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <div class="p-section--hero">
+    <div class="row">
+      <div class="col-8">
+        <h1 class="p-heading--1 u-no-margin--bottom">Ubuntu Summit 2024</h1>
+        <h2 class="p-heading--2">October 25-27</h2>
+        <h3 class="p-heading--5">The Hague, Netherlands</h3>
+        <p>
+          The Ubuntu Summit is a showcase for the innovative and the ambitious. We invite all experts, builders, engineers, tinkerers, designers and dreamers to join us for an unforgettable three days.
+        </p>
+        <p>
+          <a href="https://events.canonical.com/event/51/registrations/"
+             class="p-button--positive">Register for free</a>
+        </p>
+      </div>
+      <div class="col-4 u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/9e91caf9-Canonical%20Ubuntu%20Summit%20Logo%202024%20with%20text.svg"
+             width="350"
+             alt="Ubuntu Summit 2024 Logo" />
```

Comment:
I would suggest using the image tag to optimise this image.

Reply:
Ah I see, you mean using jinja image tag?

`{{ image(url="https://assets.ubuntu.com/v1/d3039f38-MAAS-logo.svg",
                                alt="MAAS",
                                width="104",
                                height="42",
                                hi_def=True,
                                loading="auto") | safe}}`